### PR TITLE
library/cluster_vm: add new commands to add / remove Pacemaker remotes (Yocto)

### PR DESCRIPTION
**Pull request by dupremathieu**
This commit adds a new command to the cluster_vm module to add a Pacemaker remote. This command requires a VM name, a remote name, a remote address and can optionally take a remote port and a remote timeout.

This commit also adds a new command to the cluster_vm module to remove a Pacemaker remote. This command requires a VM name.

### CODE REVIEW

File: library/cluster_vm.py [hunk 1]
```diff
@@ -67,6 +67,10 @@
         arguments I(name), I(metadata_name)
       - C(define_colocation) Define colocation rules. Require
         arguments I(name), I(colocated_vms)
+      - C(add_pacemaker_remote) Add a pacemaker remote. Require arguments
+        I(name), −I(remote_name), I(remote_address)
+      - -C(remove_pacemaker_remote) Remove a pacemaker remote. Require arguments
+         I(name)
     choices: [ create, remove, list_vms, start, status, stop, clone,
```
Comment (eroussy) [inline library/cluster_vm.py:74]: Should'nt you add "add_pacemaker_remote" and "remove_pacemaker_remote" in the choices ?

Comment (dupremathieu) [reply library/cluster_vm.py:74]: Done.

File: library/cluster_vm.py [hunk 2]
```diff
@@ -67,10 +67,14 @@
         arguments I(name), I(metadata_name)
       - C(define_colocation) Define colocation rules. Require
         arguments I(name), I(colocated_vms)
+      - C(add_pacemaker_remote) Add a pacemaker remote. Require arguments
+        I(name), −I(remote_name), I(remote_address)
+      - -C(remove_pacemaker_remote) Remove a pacemaker remote. Require arguments
+         I(name)
```
Comment (Revalioli) [inline library/cluster_vm.py:73]: Remove extra `-` on line 72, and extra indentation in line 73

File: library/cluster_vm.py [hunk 3]
```diff
@@ -210,7 +214,27 @@
...
+      - This parameter is required if I(command) is C(add_pacemaker_remote)
+    type: str
+  remote_port:
+    description:
+      - Port of the remote to add
+      - This parameter is optional if I(command) is C(add_pacemaker_remote)
+    type: int
+  remote_timeout:
+    description:
+      - Timeout of the remote to add
```
Comment (Revalioli) [inline library/cluster_vm.py:235]: Specify unit of timeout (seconds if I'm not mistaken ?)

**Review by Revalioli — CHANGES_REQUESTED** [review]

File: library/cluster_vm.py (hunk 2 above)
Comment (dupremathieu) [reply library/cluster_vm.py:73]: Done

**Review by eroussy — APPROVED** [review]

**Review by Revalioli — APPROVED** [review]

**Pull request merged**